docs/developer/stock-allocation.mdx

```diff
@@ -148,7 +148,11 @@ the [`updateWarehouse`](api-reference/products/mutations/update-warehouse.mdx) m
 There is an option to change the following warehouse fields: `name`, `slug`, `email`, `address` and
 also define privacy and `click and collect` options.
 
-We can define warehouse as public or private. Public warehouse allows you to query the stock information publicly in the API. Private warehouse requires permission.
+#### Public vs. private warehouses
+
+We can define warehouse as public or private. Public warehouse allows you to query the stock information publicly in the API. Private warehouse requires permission. The [`quantityAvailable`](api-reference/products/objects/product-variant.mdx#productvariantquantityavailableint--) field on `ProductVariant` type is the sum of the available quantity in all public and private warehouses.
+
+Storefronts can display product availability at specific locations with public stock. To protect business-sensitive details, you can cap public stock at the maximum orderable quantity per checkout: [`quantityLimitPerCustomer`](api-reference/products/inputs/product-variant-input.mdx#productvariantinputquantitylimitpercustomerint--). Therefore, if a warehouse holds 999 pieces and the max orderable quantity is 50, the API will return 50 as the available stock.
```

Comment:
Theres the link for product input, but this setting can be changed globally in SiteSettings (and adjusted in linked variant input).